src/ui/actions/session.ts

```diff
@@ -198,9 +198,10 @@ export function createSocket(
       const experimentalSettings: ExperimentalSettings = {
         // Uncomment this to get a new session every time you refresh
         // controllerKey: String(Math.floor(Math.random() * 10e8)),
-        listenForMetrics: !!prefs.listenForMetrics,
+        cacheScanData: true,
```

Comment:
Thanks for that comment @bvaughn! The setting was *supposed* to be here! But I forgot to thread it through 💀

Reply:
Gotcha. So whatever value you have set at the start of your session gets passed to the API. If you change it, you'd have to reload for it to have an impact?

Reply:
Yep, if you change it you'll have to refresh, and because your session settings have changed you will automatically get a new session.